test/lifecycle/error.test.ts

```diff
@@ -272,4 +272,84 @@ describe('error', () => {
 			somePretty: 'json'
 		})
 	})
+
+	describe('handle scoped error', async () => {
```

Comment:
@SaltyAom Please, can you confirm if these tests define the expected behavior when handling locally the error?